tests/unit_tests/test_run_path_creation.py

```diff
@@ -194,6 +194,33 @@ def test_run_template_replace_in_ecl_data_file(key, expected, prior_ensemble):
     ) == f"I WANT TO REPLACE:{expected}"
 
 
+def test_that_error_is_raised_when_data_file_is_badly_encoded(prior_ensemble):
+    config_text = dedent(
+        """
+    NUM_REALIZATIONS 1
+    ECLBASE ECL_CASE<IENS>
+    DATA_FILE MY_DATA_FILE.DATA
+    """
+    )
+    Path("MY_DATA_FILE.DATA").write_text("I WANT TO REPLACE:<DATE>", encoding="utf-8")
+    Path("config.ert").write_text(config_text, encoding="utf-8")
+
+    ert_config = ErtConfig.from_file("config.ert")
```

Comment:
This looks like a good improvement, but as we already know about this file when ert starts, perhaps we should validate it there? Then the user would get a message in the Suggester instead of when they start the model. We could of course do both in case they change it after ert has started.

Reply:
I think we already do that.

See this test: https://github.com/equinor/ert/blob/5072a4eb4c17a20d45efa81df191c5728518d5fe/tests/unit_tests/config/test_ert_config.py#L423-L451

Are you perhaps thinking of something else?

Reply:
Ahh, I must have missed it, sorry.

Reply:
Come to think of it, why does not your test raise the config error when you create `ErtConfig`?